user_guide/intro_navigation.rst

```diff
@@ -0,0 +1,110 @@
+.. _user-guide-getting-started-controls:
+  
+Navigation Basics
+=================
+
+OroCRM provides a broad set of navigation tools that allow you to easily access necessary information and tools. The 
+system helps to ensure that the frequent actions can be performed with minimal effort by keeping required information at 
+hand. This means no important updates are missed.
```

Comment:
Edit: OroCRM provides a broad set of navigation tools that allow you to easily access necessary information and tools. The system helps to ensure that frequent actions can be performed with minimal effort by keeping required information at hand. This means no important updates are missed.